10-JavaScript/welcome-to-js.md

```diff
@@ -67,18 +67,30 @@ undefined
 >
 ```
 
-__Hint: Use `.exit` to get out of the Node REPL__
+__Hint: Use `.exit` or ctrl-d to get out of the Node REPL__
 
 Now that we've got a REPL available to us, let's jump in with some JavaScript specifics. We are going to talk about __variables__, __datatypes__, and __functions__.
 
 ### Variables
-__Declare all variables with the var operator!__
+__Declare all variables with the var or let operators!__
 
 ```javascript
 var five = 5;
+let six  = 6;
 ```
 
-If you omit `var` you will get a __global__ variable, which can lead to all sorts of problems. __JUST DON'T DO IT!__
+If you omit `var` or `let` you will get a __global__ variable, which can lead to all sorts of problems. __JUST DON'T DO IT!__
+
+`var` and `let` mean subtly different things, both having to do with [scope](https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var-to-declare-a-variable) which we'll get to later.  A good rule of thumb is to use `let` unless you are working with an older version of the JavaScript interpreter.
+
+#### Constants
+
+You can declare a constant with the `const` operator.  Constants are like variables declared with `let`, but cannot change their value through reassignment.
+
+```javascript
+const FIVE = 5;
+FIVE = 6;  // TypeError: Assignment to constant variable.
+```
```

Comment:
Maybe add a note about "commonly we define functions and classes using `const`, because these are things that shouldn't ever be reassigned".

Reply:
Added in the fun with functions notes.